jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -48,6 +48,9 @@ WITH_OPENSSL            := @WITH_OPENSSL@
 ## DDR
 OPENJ9_ENABLE_DDR       := @OPENJ9_ENABLE_DDR@
 
+## JITSERVER
+OPENJ9_ENABLE_JITSERVER    := @OPENJ9_ENABLE_JITSERVER@
```

Comment:
Can you please remove the extra `#` here (and above for DDR), and remove the extra spaces before `:=`.